pilot/docker/envoy_policy.yaml.tmpl

```diff
@@ -123,6 +123,59 @@ static_resources:
         require_client_certificate: true
 {{- end }}
     name: "15004"
+  - address:
```

Comment:
15004 and 9091 listeners only differ in tls configuration.
If they differ in more ways, that is hard to see here. 

Can you abstract that part out and use `template` keyword to render it ?

Reply:
Not sure it's a good idea to create functions in templates... This is deliberately "flat" to avoid yet another way to produce templates.

Reply:
Let's delay refactoring until after we normalize templates across helm, this, and sidecar bootstrap. There's just too many ways to do the same thing.